cmd/security-agent/app/app.go

```diff
@@ -110,6 +108,7 @@ func start(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("unable to set up global agent configuration: %v", err)
 	}
+
 	// Setup logger
 	syslogURI := coreconfig.GetSyslogURI()
 	logFile := coreconfig.Datadog.GetString("log_file")
```

Comment:
unrelated, and can be addressed in separate PR, but the security-agent should log to a different file than the core agent. Log rotations and such that are handled by the logger will likely not work properly otherwise

Reply:
Indeed, will do another PR for that